waku/node/peer_manager/peer_manager.nim

```diff
@@ -50,10 +53,10 @@ const
   BackoffFactor = 4
 
   # Limit the amount of paralel dials
-  MaxParalelDials = 10
+  MaxParallelDials = 10
 
   # Delay between consecutive relayConnectivityLoop runs
-  ConnectivityLoopInterval = chronos.seconds(15)
+  ConnectivityLoopInterval = chronos.minutes(1)
```

Comment:
>Changing ConnectivityLoopInterval from 15 seconds to 1 minute was one of the causes of failed tests and messages not being properly relayed right after a node starts running.

> What's the reason behind this change? Performance? Does it make sense for it to be configurable?
We should definitely check if this change keeps causing problems during startup and cause failed tests because of not properly discovering peers.

1m was chosen to allow dialing many more peers than the previous peer manager. It seams long enough.

As for the tests, waku-simulator has the quirk that ALL nodes dial ALL other nodes at the SAME time, which would never happen in reality. I randomized the selection of peers to speed up connections in the simulator.

As for making it dynamic, yes we could but it would not speed up Discv5 and we already connect to all the suitable nodes we know every 1m. It seams fine to me.